crates/sqlexec/src/remote/planner.rs

```diff
@@ -45,13 +45,13 @@ impl QueryPlanner for RemoteLogicalPlanner {
 #[derive(Debug, Clone)]
 pub struct RemotePhysicalPlanner {
     /// Client to remote services.
-    remote_client: RemoteSessionClient,
+    _remote_client: RemoteSessionClient,
```

Comment:
Why not remove this?

Reply:
The planner stuff is going to change quite a bit. I'm working on splitting up session context here https://github.com/GlareDB/glaredb/pull/1589 because I was getting confused about what's execution where. By splitting the session context, it'll be easier to tell when we need a custom planner (and it'll probably clean up some other confusing logic too).